js/Readium.js

```diff
@@ -82,6 +82,8 @@ define(['readium_shared_js/globals', 'text!version.json', 'jquery', 'underscore'
         console.log("EPUB doc base href:");
         console.log(baseHref);
 
+        var style = "<style>img {-webkit-touch-callout: none; -webkit-user-select: none; user-select: none;}</style>";
```

Comment:
The issue with this is that we block the download functionnality even for unprotected books.